Can't webpack PnP Starter Intranet for Office 365/SharePoint Online

#### Category
[ x ] Bug
[ ] Enhancement

#### Environment
[ x ] Office 365 / SharePoint Online
[ ] SharePoint 2016
[ ] SharePoint 2013

I was installing PnP Starter Intranet for Office 365/SharePoint Online by following the instructions in the md file. (https://github.com/SharePoint/PnP/tree/master/Solutions/Business.StarterIntranet).
When I reached the webpack step: "Check if everything is OK by running the "webpack" cmd from the ".\app" folder. You shouldn't see any errors here.", it should have not thrown any errors when the webpack command is run.

**Observed Behavior**

SharePointPnPPowerShellOnline version 2.14.1704.0 (April)
Node.js 6.10.2
webpack version: 1.14.0
typescript version: 2.2.0
sp-pnp-js version 1.0.6
Windows Server 2012 R2

The script starts with a message about ts-loader and then there are many "error TS2304: Cannot find name 'Promise'." error messages.

PS C:\projects\pnp-master\solutions\business.starterintranet\app> webpack
webpack : ts-loader: Using typescript@2.2.0 and C:\projects\pnp-master\solutions\business.starterintranet\app\tsconfig.json
At line:1 char:1
+ webpack
+ ~~~~~~~
    + CategoryInfo          : NotSpecified: (ts-loader: Usin...p\tsconfig.json:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
Hash: 8c5c3c003d2e360e124e
Version: webpack 1.14.0
Time: 22687ms
                     Asset       Size  Chunks             Chunk Names
               layouts.css    1.07 kB       0  [emitted]  app
         ./img/spinner.gif  420 bytes          [emitted]  
   ./img/logo_intranet.png    16.8 kB          [emitted]  
./img/favicon_intranet.ico    1.15 kB          [emitted]  
                 js/app.js     622 kB       0  [emitted]  app
              js/vendor.js    1.48 MB       1  [emitted]  vendor
                portal.css    14.9 kB       0  [emitted]  app
         bootstrap-iso.css     134 kB       0  [emitted]  app
   ./img/default_image.png  762 bytes          [emitted]  
          layouts-edit.css  805 bytes       0  [emitted]  app
             js/app.js.map     715 kB       0  [emitted]  app
            portal.css.map  929 bytes       0  [emitted]  app
     bootstrap-iso.css.map     160 kB       0  [emitted]  app
           layouts.css.map  108 bytes       0  [emitted]  app
      layouts-edit.css.map  103 bytes       0  [emitted]  app
          js/vendor.js.map    1.73 MB       1  [emitted]  vendor
   [0] multi vendor 124 bytes {1} [built]
    + 292 hidden modules

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(10,31): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(14,36): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(20,39): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(26,38): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(32,41): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(38,41): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(54,45): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(60,42): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(65,28): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(72,73): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(78,40): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(85,54): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(91,46): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(97,57): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(102,32): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(106,27): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(112,55): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\userprofiles.d.ts
(118,37): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\site.d.ts
(31,23): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\site.d.ts
(37,51): error TS2304: Cannot find name 'Promise'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\sp-pnp-js\lib\sharepoint\rest\site.d.ts
(43,52): error TS2304: Cannot find name 'Promise'.

**It goes on and on like this... Farther down, some of these...**

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\@types\core-js\index.d.ts
(39,27): error TS2304: Cannot find name 'IterableIterator'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\@types\core-js\index.d.ts
(47,36): error TS2304: Cannot find name 'Iterable'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\@types\core-js\index.d.ts
(226,38): error TS2304: Cannot find name 'IterableIterator'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\@types\core-js\index.d.ts
(231,35): error TS2304: Cannot find name 'IterableIterator'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\@types\core-js\index.d.ts
(236,37): error TS2304: Cannot find name 'IterableIterator'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\@types\core-js\index.d.ts
(349,33): error TS2304: Cannot find name 'IterableIterator'.

ERROR in C:\projects\pnp-master\solutions\business.starterintranet\app\node_modules\@types\core-js\index.d.ts
(350,31): error TS2304: Cannot find name 'IterableIterator'.

--- a/Solutions/Business.StarterIntranet/app/tsconfig.json
+++ b/Solutions/Business.StarterIntranet/app/tsconfig.json
@@ -9,6 +9,6 @@
       "knockout",
       "knockout.mapping"
     ],
-    "lib": ["dom", "es5"] //Include some usefull librairies https://www.typescriptlang.org/docs/handbook/compiler-options.html
+    "lib": ["dom", "es5", "es2015"] //Include some usefull librairies https://www.typescriptlang.org/docs/handbook/compiler-options.html
   }
 }

